R/df_label.R

```diff
@@ -40,8 +40,8 @@
 #'
 #' adsl <- xportr_df_label(adsl, metadata, domain = "adsl")
 xportr_df_label <- function(.df,
-                            metadata = NULL,
-                            domain = NULL,
+                            metadata = attr(.df, "_xportr.df_metadata_"),
+                            domain = attr(.df, "_xportr.df_arg_"),
```

Comment:
What do you folks feel about changing the default parameter value from `NULL` to the `.df` attribute.

Be aware that `attr` always returns NULL if it doesn't exist of if the first parameter is NULL/NA

``` r
attr(NULL, "_xportr.df_metadata_")
#> NULL
attr(NA, "_xportr.df_metadata_")
#> NULL
```

<sup>Created on 2023-12-15 with [reprex v2.0.2](https://reprex.tidyverse.org)</sup>

Reply:
Umm... I don't know.  It looks really confusing to me to see , but I don't have a better alternative!!  @atorus-research/xportr-development-team others have thoughts?

Reply:
@averissimo I worry that exposing the attribute structure may confuse people a bit. I'd prefer to keep the NULLs personally